### ISSUE TITLE
Donkey kong 64 doesn't boot up

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jennorby/mupen64plus#273]
```
Describe your system:
 - Linux distribution: Debian
 - Machine type: 64-bit
 - Mupen64Plus version: svn1404 + bug 51 related patches
 - Plugins used: glide64

For visual artifacts, give:
 - MD5 118E9CE360B97A6F8DAB2C9F30660BF5
 - Donkey Kong 64 (E) [!]

Describe the problem:
It seems that the command at address 0x80603fb0 or following command (in
delay slot) isn't interpreted correctly and it causes Donkey Kong 64 not to
boot up correctly. This is a 'bne  $at, $r0, 0x80603FA4'

It seems to be inside a loop "forever" near that position. The code looks
like this:

80603FA4 lw    $t7, 0x0000($v0)
80603FA8 addiu $v0, 0x0004
80603FAC sltu  $at, $v0, $a0
80603FB0 bne   $at, $r0, 0x80603FA4
80603FB4 addu  $v1, $v1, $t7
80603FB8

And after the next round it will do an exception with cause 0x10000400.

I cannot upload it here as "project attachment quota exceeded".
http://www.mediafire.com/?dldyazngymn



```

Original issue reported on code.google.com by `sven@narfation.org` on 26 Sep 2009 at 11:29

**Comment by GoogleCodeExporter** [comment]
```
Sry, this savegame is exact one round before the actual interessting bne. It 
seems
that the interrupt by itself is the problem. Somehow.
```

Original comment by `sven@narfation.org` on 27 Sep 2009 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
Pokemon Puzzle League 000364BAC80E41D9060A31A5923874B7 doesn't boot up either 
and
also creates many, many, many SP_INT interrupts instead. Maybe this is related 
somehow.
```

Original comment by `sven@narfation.org` on 28 Sep 2009 at 10:52

**Comment by GoogleCodeExporter** [comment]
```
bump

I really want to play DK on my mac yall
```

Original comment by `firstdog...@gmail.com` on 5 Oct 2009 at 8:04

**Comment by GoogleCodeExporter** [comment]
```
srsly

i <3 this game

but i havent played in 6 years

help a dog out
```

Original comment by `firstdog...@gmail.com` on 5 Oct 2009 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
what the petes sake bro

its been like 18 hours

somebody help me
```

Original comment by `firstdog...@gmail.com` on 5 Oct 2009 at 11:29

**Comment by GoogleCodeExporter** [comment]
```
Bugs doesn't get fixed by writing stupid entries to a bug tracker.
```

Original comment by `sven@narfation.org` on 6 Oct 2009 at 8:10

**Comment by GoogleCodeExporter** [comment]
```
Corrected savestate http://www.mediafire.com/?0jbfwixgmyi
```

Original comment by `sven@narfation.org` on 8 Oct 2009 at 6:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `richard...@gmail.com` on 12 Jan 2010 at 10:48
- Added labels: **Component-Core**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Don't take my comment too seriously, because I just don't know, but may this be
caused by some division-by-zero strange behaviour? In the Mupen64Plus version 
1.5 I
had, I got some division-by-zero messages just after the Rare logo (IIRC).
Unfortunately, after I compiled from the source, I didn't get those errors 
anymore
for some strange reason.

But I tried adding some checks to DIV_S for nearly-zero values, and at almost 
the end
of the intro I get one place where it's very close to zero, twice.

DIV_S 0.000003 @ 806545CC

I don't know if Mupen64Plus is accurate in floating point calculations, so if it
isn't, that might be an interesting place to look.
```

Original comment by `GameZel...@gmail.com` on 17 May 2010 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
This rom just needs to use 4k eeprom. I will try to merge the wii64 stuff with
mupen64plus.ini later
```

Original comment by `sven@narfation.org` on 3 Jun 2010 at 9:14

**Comment by GoogleCodeExporter** [comment]
```
This specific problem should be fixed in 196:24b506da4250
```

Original comment by `sven@narfation.org` on 3 Jun 2010 at 10:41

**Comment by GoogleCodeExporter** [comment]
Original comment by `richard...@gmail.com` on 4 Jun 2010 at 2:04
- Changed state: **Fixed**
